action/init.go

```diff
@@ -13,9 +13,9 @@ const (
 	SEND Type = 0x01
 
 	//staking related transaction
-	APPLYVALIDATOR Type = 0x11
```

Comment:
Since we keep committing new changes to develop branch, It's good for you to keep your branch sync with develop because conflicts could be painful in final merge.

Reply:
No pain of rebasing 5-10 minutes took previous time. APPLYVALIDATOR works 50 vs 50, so in some execution the node could stop after this action.